libs/utils/test/test_sstream.cpp

```diff
@@ -96,11 +96,6 @@ TEST(sstream, Formatting) {
         ss << (bool) true;
         EXPECT_STREQ("1", ss.c_str());
     }
-    {
```

Comment:
This behavior is platform-specific (i.e., Windows outputs leading zeros without the 0x), and IMO not worth the upkeep of a unit test